common/src/main/java/org/opensearch/ml/common/transport/sync/MLSyncUpInput.java

```diff
@@ -35,6 +36,8 @@ public class MLSyncUpInput implements Writeable {
     // profile API has consistent data with model index.
     private Map<String, Boolean> deployToAllNodes;
 
+    public static final Version MINIMAL_SUPPORTED_VERSION_FOR_MODEL_TTL = Version.V_2_14_0;
```

Comment:
`MINIMAL_SUPPORTED_VERSION_FOR_MODEL_TTL` is more related with `common/src/main/java/org/opensearch/ml/common/model/MLDeploySetting.java`, can we move to `MLDeploySetting` ?

Reply:
Updated in the newest commit